pam_ussh.go

```diff
@@ -68,6 +70,10 @@ func pamLog(format string, args ...interface{}) {
 // authenticate validates certs loaded on the ssh-agent at the other end of
```

Comment:
warning: cyclomatic complexity 25 of function authenticate() is high (> 15) (gocyclo)